includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -122,9 +122,9 @@ function perflab_aao_autoloaded_options_size() {
  *
  * @global wpdb $wpdb WordPress database abstraction object.
  *
- * @return array Autoloaded data as option names and their sizes.
+ * @return array<string, object{option_name: string, option_value_length: int}> Autoloaded data as option names and their sizes.
```

Comment:
This was wrong. It is not an associative array. It should rather have been:

```diff
- * @return array<string, object{option_name: string, option_value_length: int}> Autoloaded data as option names and their sizes.
+ * @return array<object{option_name: string, option_value_length: int}> Autoloaded data as option names and their sizes.
```

Reply:
See https://github.com/WordPress/performance/pull/1179#pullrequestreview-2041624276